aea/aea_builder.py

```diff
@@ -1300,6 +1300,11 @@ def _load_and_add_components(
             if configuration in self._component_instances[component_type].keys():
                 component = self._component_instances[component_type][configuration]
             else:
+                if component_type == ComponentType.SKILL:
+                    is_skill_abstract = cast(SkillConfig, configuration).is_abstract
+                    if is_skill_abstract:
+                        load_aea_package(configuration)
+                        continue
```

Comment:
very ugly, but I think it's the best place where to do this check. `Skill.from_config` should still return a component instance, even if abstract. Thoughts?

Reply:
At least extract it as a generic helper function so we can then extend it for other components, something like:
```
if ...
elif is_abstract_component(configuration):
    load_aea_package(configuration)
else:
    ....
```
where `is_abstract_component` is a helper function in helpers